.github/workflows/ci.yml

```diff
@@ -53,3 +53,38 @@ jobs:
 
       - name: Run tests
         run: pnpm coverage
+
+      # - name: Setup TeX Live
+      #   uses: teatimeguest/setup-texlive-action@v3
+      #   with:
+      #     packages: scheme-basic
+
+      - name: Install texlive-latex
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y texlive-latex-base texlive-xetex texlive-latex-extra texlive-fonts-recommended texlive-fonts-extra
+          xetex -v
+
+      - name: Install fonts
+        run: |
+          sudo apt install fonts-linuxlibertine
+          sudo apt install fonts-noto-cjk fonts-noto-cjk-extra
+          sudo apt install latex-cjk-all
```

Comment:
`latex-cjk-all` is included in `texlive-lang-all` package, so no need here.

I am planning to support more languages other than CJK, so let us just use `texlive-lang-all`.